lib/browser/storage.js

```diff
@@ -5,10 +5,18 @@ try {
   hasStorage = "localStorage" in window;
   // Attempt to access localStorage
   localStorage.length;
+
+  // Attempt to store and read entries from the local storage to detect Private
+  // Mode on Safari on iOS (see #49)
+  var test = 'tusSupport';
+  localStorage.setItem(test, test);
+  localStorage.removeItem(test);
```

Comment:
Also, I have the feeling that the probability that an attribute with `tusSupport` is used by another application is very small.

Alternatively, we could use something like `'tusSupport_dHVzU3VwcG9ydA=='`